wpimath/src/main/native/include/frc/kinematics/DifferentialDriveOdometry.h

```diff
@@ -27,24 +27,37 @@ class WPILIB_DLLEXPORT DifferentialDriveOdometry {
   /**
    * Constructs a DifferentialDriveOdometry object.
    *
+   * IF leftDistance and rightDistance are unspecified, 
+   * You NEED to reset your encoders (to zero).
+   * 
    * @param gyroAngle The angle reported by the gyroscope.
+   * @param leftDistance The distance traveled by the left encoder.
+   * @param rightDistance The distance traveled by the right encoder.
    * @param initialPose The starting position of the robot on the field.
    */
   explicit DifferentialDriveOdometry(const Rotation2d& gyroAngle,
+                                     const units::meter_t leftDistance = 0_m,
+                                     const units::meter_t rightDistance = 0_m,
```

Comment:
const is redundant for value types, since they're copied.

Reply:
these `const` annotations have been removed.